### PR TITLE
MQTTNIO proposal

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: proposals/0018-mqtt-nio.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+```
+Whoever subscribes to the "JSONTest" topic with a v5.0 client will also receive the `contentType` property along with the payload.
+
+## Maturity Justification
+
+Sandbox. The package has one author. It is fairly new, v1.0.0 was released a year ago.  
+
+## Alternatives considered
+
+There are two other major Swift MQTT client implementations [CocoaMQTT](https://github.com/emqx/CocoaMQTT) and  [SwiftMQTT](https://github.com/aciidb0mb3r/SwiftMQTT). Neither of these are built on top of Swift NIO or support v5 of the protocol.
```
Comment (0xTim) [inline proposals/0018-mqtt-nio.md]: Looks like CocaoMQTT now supports v5 of the protocol. However it's still not built on NIO or aimed at the server ecosystem, and the SSWG has an explicit aim to allow multiple packages for the same thing

**Review by 0xTim — APPROVED** [review]
+1 from me. Proposal looks good

File: proposals/0018-mqtt-nio.md (hunk 1 above)
Comment (adam-fowler) [reply proposals/0018-mqtt-nio.md]: Updated

File: proposals/0018-mqtt-nio.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+# MQTTNIO
+
+* Proposal: [SSWG-0018](0018-mqtt-nio.md)
+* Author: [Adam Fowler](https://github.com/adam-fowler)
+* Review Manager: TBD
```
Comment (0xTim) [inline proposals/0018-mqtt-nio.md]:
```suggestion
* Review Manager: [Tim Condon](https://github.com/0xTim)
```

**Review by 0xTim — COMMENTED** [review]
Add me as a review manager

**Pull request merged**